src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/CombinationController.php

```diff
@@ -707,6 +719,12 @@ private function getErrorMessages(Exception $e): array
                 'The object cannot be loaded (or found)',
                 'Admin.Notifications.Error'
             ),
+            CannotGenerateCombinationException::class => [
+                CannotGenerateCombinationException::DUE_TO_DIFFERENT_ATTRIBUTES_BETWEEN_SHOPS => $this->trans(
+                    'Attributes must be associated with every store to generate a combination for all stores',
```

Comment:
A new wording here. Ping @PrestaShop/wording-managers for adjustments.
The case - when we try to generate combinations in all shops, but the attributes between shops are different, then this error is thrown.

Reply:
```suggestion
                    'Selected attributes must be associated with every store to generate a combination for all stores',
```

Reply:
Alternative
```suggestion
                    'You cannot generate combinations for all stores if the selected attributes are not associated on all stores',
```

Reply:
still waiting for @PrestaShop/wording-managers decision approval before changes (btw the second suggestion doesn't sound well with the "on all stores") 😄